prototype2/efu/HwCheck.cpp

```diff
@@ -93,6 +93,7 @@ void HwCheck::debugPrint(struct ifaddrs * ifa) {
 }
 
 
+#if 0
 int main(int argc, char * argv []) {
```

Comment:
Seeing as this code is basically commented out, I think it should be removed. One alternative is to add it to a unit test.

Reply:
yes, I forgot to delete the code. It started out as a standalone program for testing